test/hooks/achievement-rule-checker.test.js

```diff
@@ -249,4 +266,106 @@ describe('\'achievement-rule-checker\' hook', () => {
     });
     assert.deepEqual(result2[0].amount, 1);
   });
+
+  it('Gives achievement with logical amount >', async () => {
+    await app.service('events').create({
+      'name': 'EventGiving10XP',
+      'user_id': user_id
+    });
+
+    let moreResult1 = await app.service('achievements').find({
```

Comment:
All of these should be `const`.

Reply:
We probably should consider using the `prefer-const` rule of eslint.

Reply:
But till we do that, this will be fixed once the test utils #55 are merged anyways.